docs/topics/mta-what-is-the-toolkit.adoc

```diff
@@ -13,20 +13,23 @@
 = About the {ProductName}
 
 [discrete]
-== What is the {ProductName}?
+== What is the {ProductName}
 
-{ProductName} ({ProductShortName}) accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
+{ProductName} ({ProductShortName}) accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user or command line interfaces.

```

Comment:
💡 RedHat.TermsSuggestions 1(344-347) Depending on the context, consider using 'through', 'by', 'from', 'on', or 'by using' rather than 'via'.